shell/edit/cloudcredential.vue

```diff
@@ -272,47 +274,46 @@ export default {
 </script>
 
 <template>
-  <form class="filled-height">
-    <Loading v-if="$fetchState.pending" />
-    <CruResource
-      v-else
+  <Loading v-if="$fetchState.pending" />
+  <CruResource
+    v-else
+    :done-params="$attrs['done-params'] /* Without this, changes to the validationPassed prop end up propagating all the way to the root of the app and force a re-render when the input becomes valid. I haven't found a reasonable explanation for why this happens. */"
```

Comment:
The actual fix

Reply:
Would it be appropriate to do `v-bind="$attrs"` instead? It's hard to tell, but it seems like all the fallthrough attributes are intended for this node.

If not, I think that it would be more clear if we created a `doneParams` prop for this component. This way, we could have a more explicit contract for this component instead of relying on implicit attributes.

Reply:
we had an out of band discussion and decided to keep the implementation as it is in this PR. There's too many conflicts and edge cases that complicate making a change.